README.md

```diff
@@ -0,0 +1,175 @@
+### Creating the DUH document
+NOTE: this tutorial was made using version 1.15.0 of DUH
+
+The DUH document is a json5 file that describes an IP block similar to IPXACT.
+The DUH json schema is defined [here](https://github.com/sifive/duh-schema).
+DUH is also a suite of tools that help users author DUH documents and generate
+useful artefacts from them. This section describes how to use some of the DUH
+tools to author a DUH document for the PIO block.
+
+First you will need to install [DUH](https://github.com/sifive/duh). To install
+DUH in your current directory run
+```bash
+npm i duh
```

Comment:
show how to pull a specific version

Reply:
I would agree that if we're telling people that these instructions are for DUH 1.15.0, we might as well pin that version here, especially for people who may not be familiar with how to use `npm` to grab a specific version of a library.